damien/api/department_controller.py

```diff
@@ -71,3 +73,34 @@ def update(department_id):
         return tolerant_jsonify(department.to_api_json())
     else:
         raise ResourceNotFoundError(f'Department {department_id} not found.')
+
+
+@app.route('/api/department/<department_id>/contact', methods=['POST'])
+@login_required
+def update_contact(department_id):
+    if not current_user.is_admin:
+        raise ForbiddenRequestError('Admin required.')
```

Comment:
@lyttam - would a `@admin_required` annotation make sense

Reply:
@johncrossman Yes, good catch. I will change it in the next PR.